Source/SwiftLintFramework/Extensions/File+Hashable.swift

```diff
@@ -0,0 +1,18 @@
+import SourceKittenFramework
+
+extension File: Hashable {
```

Comment:
We should do this in SourceKittenFramework itself.

Reply:
I liked your implementation so pasted it verbatim in a PR towards SourceKitten and credited you in the changelog: https://github.com/jpsim/SourceKitten/pull/594

Thanks!

Reply:
And a new release has been cut: https://github.com/jpsim/SourceKitten/releases/tag/0.23.2